tests/e2e/csi_static_provisioning_basic.go

```diff
@@ -204,14 +205,12 @@ var _ = ginkgo.Describe("Basic Static Provisioning", func() {
 		pvc = nil
 
 		ginkgo.By("Verify PV should be deleted automatically")
-		framework.ExpectNoError(framework.WaitForPersistentVolumeDeleted(client, pvName, poll, pollTimeoutShort))
+		framework.ExpectNoError(framework.WaitForPersistentVolumeDeleted(client, pvName, poll, supervisorClusterOperationsTimeout))
```

Comment:
Where is the `supervisorClusterOperationsTimeout` variable declared ?
I cannot find it in the `tests/e2e/e2e_common.go` ?

Reply:
`supervisorClusterOperationsTimeout` is an existing  variable